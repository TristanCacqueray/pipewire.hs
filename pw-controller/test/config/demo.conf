# A config file

when device:r"Sennheiser USB headset"

  # Ensure laptop mic is not used
  disconnect output device:"sof-hda-dsp"
