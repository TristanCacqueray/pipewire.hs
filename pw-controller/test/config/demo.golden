Right
    [ When
        { condition = DeviceMatcher "Sennheiser USB headset"
        , body =
            [ Disconnect
                { direction = Output
                , target = Target
                    { kind = Device
                    , matcher = re"^sof-hda-dsp$"
                    }
                }
            ]
        }
    ]